### ISSUE TITLE
Include exit code in accessible buffer for failed commands

### ISSUE THREAD

**Issue by vscodenpa** [vscodenpa/testissues#16963]
Sighted users can determine if a command failed via the color of or by hovering the command decoration. That's tedious for a screen reader user, so we should include this info in the accessible buffer.
